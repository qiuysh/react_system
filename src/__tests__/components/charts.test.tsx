/**
 * /*
 *
 * @format
 * @Author: wulin
 * @Date: 2021-02-24 15:54:21
 * @LastEditors: wulin
 * @LastEditTime: 2021-04-28 15:15:42
 * @Description: This is desc
 * @FilePath: /react_system/src/__tests__/components/charts.test.tsx
 */

/** @format */
import * as React from "react";
import { shallow } from "enzyme";
// import "jest-canvas-mock";
// import render from "react-test-renderer";
import Bar from "@components/charts/bar";

describe("图表组件测试", () => {
  it("文本组件 -- 渲染", () => {
    const wrapper = shallow(<div>文本测试</div>);
    expect(wrapper).toMatchSnapshot();
  });

  it("bar组件 -- 渲染", () => {
    const props = {
      data: [],
      options: {
        height: 0,
        style: {},
        subTitle: "",
      },
    };
    const wrapper = shallow(<Bar {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
