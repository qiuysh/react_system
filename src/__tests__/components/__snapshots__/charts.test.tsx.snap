// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`图表组件测试 bar组件 -- 渲染 1`] = `
<ForwardRef
  data={Array []}
  xField="x"
  yField="y"
/>
`;

exports[`图表组件测试 文本组件 -- 渲染 1`] = `
<div>
  文本测试
</div>
`;
